### ISSUE TITLE
Raml parser won't accept description in body

### ISSUE THREAD

**Issue by chmac** [raml-org/raml-js-parser#130]
The second example under [responses](http://raml.org/spec.html#responses) in the spec includes a `description` inside the body. However, this RAML parser fails on that example.

I added a generic header like so:

```raml
#%RAML 0.8
---
title: e-BookMobile API
baseUri: http://api.e-bookmobile.com/{version}
version: v1

/media/{mediaid}/content:
  displayName: Content
  get:
    description: |
      Get a media file.
    responses:
      200:
        body:
          "*/*":
            description: |
                Returns the media file.
```

Then I tried to parse this like so:

```coffeescript
raml = require('raml-parser')

raml.loadFile('test.raml').then (data)->
  console.log(data)
, (error) ->
  console.log('Error parsing: ' + error)
```

Running this throws the following error:

```
Error parsing: while validating body
property: 'description' is invalid in a body
  in "test.raml", line 16, column 13
```

**Comment by chmac** [comment]
Reading the code it might be possible to use a `validate_common_properties ()` call inside `validate_body()`, but I'll leave that decision up to smarter minds than mine.

Thanks for your work on this tool by the way, it's a great help to be able to use RAML in javascript.

**Comment by dmartinezg** [comment]
This was fixed by v0.8.12
